setup.cfg

```diff
@@ -34,3 +34,8 @@ follow_imports = silent
 max-line-length = 88
 ignore = E203,E501
 
+[pydocstyle]
+# D107 Missing docstring in __init__ (reason: documented in class docstring)
+# D203 1 blank line required before class docstring (reason: pep257 default)
```

Comment:
why is this needed?

Reply:
If not added, it will require this unnecessary empty line. We don't add this extra blank line in the rest of our codebase.